lib/colorado_lottery.rb

```diff
@@ -0,0 +1,20 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = []
+  end
+
+  def interested_and_18?(name, game_name)
+    if name.game_interests == game_name && name.age > 17
```

Comment:
`name.game_interests` will return an array, which will *never* be equal to `game_name` which is a game object.  So, this method will ALWAYS return false